test/config/client/wikibase/extra-install-client.sh

```diff
@@ -6,5 +6,5 @@ set -ex
 # client
 php /var/www/html/maintenance/addSite.php --conf LocalSettings.php --wiki client_wiki  --pagepath=http://wikibase.svc/wiki/\$1  --filepath=http://wikibase.svc/w/\$1 --language en --interwiki-id my_wiki my_wiki mywikigroup
 
-# startup jobs
-php /var/www/html/maintenance/runJobs.php --wiki client_wiki --wait &
\ No newline at end of file
+# startup jobs runner internally in the wikibase-client container
+php /var/www/html/maintenance/runJobs.php --wiki client_wiki --wait &
```

Comment:
Shouldn't we have an extra job runner container here as well? Just for consistency?

Reply:
My vote is to keep it like this, but could go either way. The idea of letting this be was that the client is a "test thing", not a distribution thing. I don't yet see a problem with just running the job runner inside the Wikibase container, but moved it out to mirror what I think is a good setup in our "example". I think adding another independent job runner here might clutter things and that the test client can be more a "Swiss Army knife" thing 🤷🏼‍♂️